### ISSUE TITLE
Cannot connect to taxii Server

### ISSUE THREAD

**Issue by abhishekpokhrel** [oasis-open/cti-taxii-client#66]
When trying to connect to a public taxi server, using'Server' class of taxi2client, it gives 406 error (requests.exceptions.HTTPError: 406 Client Error: Not Acceptable for URL: https://cti-taxii.mitre.org/taxii/). Which I found out to be related to the header.
And when I used postman to connect to the same server attaching the header {'Accept': 'application/vnd.oasis.taxii+json'}... there is no problem in postman, it works.

But how do I attach the header while using taxi2client so that I can get the desired output?
![Capture12](https://user-images.githubusercontent.com/13376416/80219017-41e96600-8642-11ea-92c1-0f901705a0d0.PNG)
![python](https://user-images.githubusercontent.com/13376416/80219021-43b32980-8642-11ea-8f2d-cef2b911c778.PNG)

**Comment by emmanvg** [comment]
@abhishekpokhrel, the CTI MITRE TAXII Server is a 2.0 implementation. In order to make sure you are using the 2.0 constructs available you must import the 2.0 classes to your script/code. If you are running version [2.0.0](https://github.com/oasis-open/cti-taxii-client/releases) it will load the latest supported spec classes (TAXII 2.1). You could try:

``` python
from taxii2client.v20 import *
server = Server("https://cti-taxii.mitre.org/taxii/")
api_root = server.api_roots[0]

for collection in api_root.collections:
    print(collection.title + ":" + collection.id)
```

or 

``` python
from taxii2client import v20
server = v20.Server("https://cti-taxii.mitre.org/taxii/")
api_root = server.api_roots[0]

for collection in api_root.collections:
    print(collection.title + ":" + collection.id)
```

**Comment by abhishekpokhrel** [comment]
@emmanvg Thank you so much. This solved it.
